Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_sdio_slot0.cpp

```diff
@@ -63,19 +63,28 @@ void CWII_IPC_HLE_Device_sdio_slot0::OpenInternal()
 {
 	const std::string filename = File::GetUserPath(D_WIIROOT_IDX) + "/sd.raw";
 	m_Card.Open(filename, "r+b");
+	if (m_Card)
+		return;
+	
+	WARN_LOG(WII_IPC_SD, "Failed to open SD Card image, trying read-only access...");
+	m_Card.Open(filename, "rb");
+	if (m_Card)
+	{
+		SConfig::GetInstance().bEnableMemcardSdWriting = false;
```

Comment:
I don't think it's appropriate to change this variable in this situation. Disabling memory card writes because the SD card can't be written to is wrong (but won't cause problems often in practice), and what is going to reset the value to the default after we're done? I'd rather see a new bool variable in the CWII_IPC_HLE_Device_sdio_slot0 class that indicates that m_Card can't be written to.